Backend/AlgorithmsManager.py

```diff
@@ -177,6 +176,19 @@ def get_location_by_woeid(self, woeid):
         # TODO: possibly access the database here
         return {}
 
+    def get_all_locations(self):
+        # queryres row entry format:
+        #     woe_id, city, states, lat, long
+        queryres = self.database.query('SELECT * FROM locations;')
+        resrows = queryres.get_rows()
+        resarr = []
+        for row in resrows:
+            resarr.append(DataStructures.Location(row[0], row[1], row[2], None, row[0], row[3], row[3], row[4], row[4]).__dict__)
```

Comment:
I'm sorry about this- including max/min latitude and longitude was my idea, which ultimately didn't make sense for our project. Would you mind simplifying that to just one latitude variable and one longitude variable in the Location data structure before the client starts expecting it too?